app/src/main/java/me/phh/sip/SipMessage.kt

```diff
@@ -117,3 +117,16 @@ fun parseMessage(input: BufferedReader): SipMessage? {
 		}
 	}
 }
+
+fun serializeMessage(message: SipCommonMessage): ByteArray =
```

Comment:
I must be blind, but I can't see how you send first line -_-'

Reply:
It's the original accumulator value to the later fold, line 130.
I had it in the first fold originally as that made more sense, but that lead to a leading \r\n so I had to move it last -- thinking of fold like string join bit me...